heartbeat/stats.go

```diff
@@ -11,23 +11,23 @@ import (
 var (
 	// gauge
 	hbSubjects = prometheus.NewGaugeVec(prometheus.GaugeOpts{
-		Name: prometheus.BuildFQName("choria_stream_replicator", "replicator", "hb_subjects"),
+		Name: prometheus.BuildFQName("choria_stream_replicator", "heartbeat", "subjects_count"),
```

Comment:
Once I started documenting this I saw I screwed up when I gave you the names @ploubser so this just fixes that in line with the rest of the stat names here

Reply:
👍